h2o-py/tests/testdir_apis/H2O_Module/pyunit_h2ocluster.py

```diff
@@ -8,11 +8,7 @@ def h2ocluster():
     """
     Python API test: h2o.cluster()
     """
-
-    try:
-        h2o.cluster()
-    except Exception as e:
-        assert False, "h2o.cluster() command not is working."
+    h2o.cluster()
```

Comment:
If a function is supposed to return nothing, you can test it as:
```
ret = h2o.cluster()
assert ret is None
```

Reply:
(search for comments "no return type" in other places too)

Reply:
Done.  Gone through old tests and added them.